util/cpu/cpu_test.go

```diff
@@ -43,8 +43,12 @@ func TestCPUValue(t *testing.T) {
 	}
 	observer.Start()
 	time.Sleep(5 * time.Second)
-	require.GreaterOrEqual(t, GetCPUUsage(), 0.0)
-	require.Less(t, GetCPUUsage(), 1.0)
+	value, unsupported := GetCPUUsage()
+	require.False(t, unsupported)
+	require.GreaterOrEqual(t, value, 0.0)
+	value, unsupported = GetCPUUsage()
+	require.False(t, unsupported)
```

Comment:
```suggestion
```

Redundant test?

Reply:
compare with the origin. It is to get two values and can meet this limitation.

Reply:
refactor it.